lib/renderer/api/remote.js

```diff
@@ -175,6 +175,7 @@ const proxyFunctionProperties = function (remoteMemberFunction, metaId, name) {
       return true
     },
     get: (target, property, receiver) => {
+      if (property === 'toString') return target.toString.bind(target)
```

Comment:
Ah, I was recalling MDN docs from memory. I could have sworn it was function only, guess not =/

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy

@MarshallOfSound `target.constructor.prototype` could be another class with a custom `toString` as its property. It's best to go @kevinsawicki's route of verifying it's a method and letting JS handle the prototypical resolution